turbinia/config/turbinia_config.py

```diff
@@ -36,9 +36,6 @@
 # be deployed to.
 GCS_MOUNT_DIR = '/mnt/turbinia'
 
-# Virtualenv directory
-TURBINIAENV = '%s/turbinia-env' % HOME_DIR
```

Comment:
Did you double check this wasn't used anywhere?

Reply:
Yep, not used anywhere

Reply:
thanks.